.github/workflows/plugin-compiler-build.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Plugin-compiler build
+
+on:
+  pull_request:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  GOLANG_CROSS: 1.19-bullseye
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set tag
+        id: set-tag
+        shell: bash
+        run: |
+          echo "tag=${GITHUB_REF##*/}" >> $GITHUB_OUTPUT
+
+      - name: Configure AWS Credentials
+        id: configure-aws
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::754489498669:role/ecr_rw_tyk
+          role-session-name: ci-plugin-compiler-push
+          aws-region: eu-central-1
+
+      - name: Login to AWS ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Set docker metadata
+        id: set-metadata
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            tykio/tyk-plugin-compiler,enable=${{ startsWith(github.ref, 'refs/tags') }}
+            ${{ steps.login-ecr.outputs.registry }}/tyk-plugin-compiler
+          labels: |
+            org.opencontainers.image.title=tyk-plugin-compiler
+            org.opencontainers.image.version=${{ steps.set-tag.outputs.tag  }}
```

Comment:
Remove this as it will be set by metadata-action.

Reply:
the metadata action sets it without the `v` prefix - even if we add the v{{version}} rewrite. Goreleaser built images have it with the `v` prefix added.